apps/dotcom/sync-worker/src/TLPostgresReplicator.ts

```diff
@@ -92,27 +99,44 @@ export class TLPostgresReplicator extends DurableObject<Environment> {
 	}
 	override async alarm() {
 		this.ctx.storage.setAlarm(Date.now() + 1000)
+		this.maybeLogRpm()
+	}
+
+	private maybeLogRpm() {
+		const now = Date.now()
+		if (this.postgresUpdates > 0 && now - this.lastRpmLogTime > ONE_MINUTE) {
+			this.logEvent({
+				type: 'rpm',
+				rpm: this.postgresUpdates,
+			})
+			this.postgresUpdates = 0
+			this.lastRpmLogTime = now
+		}
 	}
```

Comment:
one of the beautiful things about the worker analytics engine is the sampling. This means we don't need to worry about how many events we write, querying should remain fast. The only concern should be pricing, but it's $0.25 per million events so 🤷🏼 if we wanted we could definitely log an event for every postgres update.

I'm happy with this implementation though, just noting that we don't need to be aggressively optimizing these.

Reply:
Sound good!